tailscale/resource_tailnet_key_test.go

```diff
@@ -67,6 +67,9 @@ func testTailnetKeyStruct(reusable bool) tailscale.Key {
 
 func setKeyStep(reusable bool, recreateIfInvalid string) resource.TestStep {
 	return resource.TestStep{
+		PreConfig: func() {
+			testServer.ResponseBody = testTailnetKeyStruct(reusable)
```

Comment:
This change just makes sure the server is ready to respond with the expected data immediately since we now do a `Read` after the `Create`.